SCons/Environment.py

```diff
@@ -961,6 +964,8 @@ def __init__(
         self.decide_source = default_decide_source
 
         self.copy_from_cache = default_copy_from_cache
```

Comment:
I left in this copy_from_cache function on the environment, although nothing is using it now (previously CacheDir was by now it uses its own class method in palce). Should these env references to copy_from_cache and copy_to_cache (new also not really needed) be removed?

Reply:
Yes. If you've obsoleted them. But make sure you add a note in RELEASE.txt and CHANGES.txt that they are gone.

Reply:
ok removed this from the environment e0a70f1